apps/emqx_lwm2m/src/emqx_lwm2m_protocol.erl

```diff
@@ -407,8 +406,8 @@ clientinfo(#lwm2m_state{peername = {PeerHost, _},
       peerhost => PeerHost,
       sockport => 5683,         %% FIXME:
       clientid => EndpointName,
-      username => null,
-      password => null,
+      username => <<>>,
```

Comment:
`undefined` is better?

Reply:
The problem now is that the auth plugins can not handle the authentication request from protocols other than MQTT, as there's no username and password fields.

This means we need to refactor the auth plugins to support multiple protocols, for now we can keep the username and password to `undefined` here.

Reply:
ty fixed